scripts/manage

```diff
@@ -0,0 +1,159 @@
+get_docker_compose_command() {
+  # Check if `docker compose` is available
+  if command -v "docker" > /dev/null && docker compose version > /dev/null 2>&1; then
+    echo "docker compose"
+  elif command -v docker-compose > /dev/null 2>&1; then
+    echo "docker-compose"
+  else
+    echo "Error: Neither 'docker compose' nor 'docker-compose' is installed." >&2
+    exit 1
+  fi
+}
+
+# Function to build Docker images
+build_docker_images() {
+  echo "Building Docker images..."
+  echo "Building traction:plugins-acapy"
+  # Check if the plugins/docker directory exists
+  if [ -d "../plugins/docker" ]; then
+    pushd ../plugins/docker > /dev/null
+    docker build -f ./Dockerfile --tag traction:plugins-acapy .. --no-cache
+    popd > /dev/null
+  else
+    echo "Error: Directory '../plugins/docker' does not exist. Check the path."
+    exit 1
+  fi
+
+  echo "Building traction:traction-agent"
+  # Check if the services/aca-py directory exists
+  if [ -d "../services/aca-py" ]; then
+    pushd ../services/aca-py > /dev/null
+    if [ -f "./Dockerfile.acapy" ]; then
+      docker build -f ./Dockerfile.acapy --tag traction:traction-agent . --no-cache
+    else
+      echo "Error: File 'Dockerfile.acapy' not found in '../services/aca-py'."
+      popd > /dev/null
+      exit 1
+    fi
+    popd > /dev/null
+  else
+    echo "Error: Directory '../services/aca-py' does not exist. Please verify the path."
+    exit 1
+  fi
+
+  # Capture the Docker Compose command
+  compose_cmd=$(get_docker_compose_command)
+
+  # Check if the command is empty or not set correctly
+  if [ -z "$compose_cmd" ]; then
+    echo "Error: Failed to determine Docker Compose command." >&2
+    exit 1
+  fi
+
+  built_services="traction-acapy-image-builder tenant-ui tenant-proxy endorser-api endorser-api-1"
+  pulled_services="ngrok-traction-agent traction-db endorser-db endorser-db-1 maildev endorser-agent endorser-agent-1"
+  other_services="traction-agent"
+
+
+ $compose_cmd build $built_services --no-cache --parallel
+ $compose_cmd pull $pulled_services
+
+ 
+ }
+
+# Function to start Docker containers
+start_docker_containers() {
+  echo "Starting Docker containers..."
+    # Capture the Docker Compose command
+    compose_cmd=$(get_docker_compose_command)
+
+    # Check if the command is empty or not set correctly
+    if [ -z "$compose_cmd" ]; then
+      echo "Error: Failed to determine Docker Compose command." >&2
+      exit 1
+    fi
+
+    # Use the captured command to build, start, or stop
+    $compose_cmd up -d
+}
+
+# Function to stop Docker containers
+stop_docker_containers() {
+  echo "Stopping Docker containers..."
+
+  # Capture the Docker Compose command
+  compose_cmd=$(get_docker_compose_command)
+
+  # Check if the command is empty or not set correctly
+  if [ -z "$compose_cmd" ]; then
+    echo "Error: Failed to determine Docker Compose command." >&2
+    exit 1
+  fi
+
+  # Use the captured command to stop and remove containers
+  $compose_cmd down
```

Comment:
`down` is a destructive operation which removes containers, volumes, and networks.  You should have a `stop` option which is non-destructive will stop the containers so they can be started again in the same state.

Reply:
Thanks! this is fixed in [63fe016](https://github.com/bcgov/traction/pull/1388/commits/63fe01684cc60f44dba90f6174ed7eff6ff5b359)